tests/test_module_ilrmsdmatrix.py

```diff
@@ -132,6 +132,25 @@ def test_ilrmsdmatrix_run(ilrmsdmatrix, mocker):
         assert lines[0] == f"A 37 38 39 40 43 44 45 69 71 72 75 90 93 94 96 132{os.linesep}"  # noqa : E501
         assert lines[1] == f"B 10 11 12 16 17 48 51 52 53 54 56 57{os.linesep}"
 
+def test_ilrmsdmatrix_run_swappedchains(ilrmsdmatrix, mocker):
```

Comment:
This one here should be in the integration folder

Reply:
fair point, moved there!